corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/query.less

```diff
@@ -17,17 +17,23 @@
   }
   .query-input-group {
     margin-right: 8px;
-    margin-bottom: 8px;
+    margin-bottom: 4px;
     margin-left: 8px;
   }
   .query-caption {
-    margin-top: 8px;
+    margin-top: 4px;
```

Comment:
this will affect all query inputs right, but just the sidebar?

Reply:
Right, all the query inputs that live anywhere within a `flex-child-sidebar` (since the selector is [nested](https://github.com/dimagi/commcare-hq/blob/f1ed6258a97d3c0348cedbe7c2df9b0be411d334/corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/query.less#L14-L27)).